src/redux/categories/categories.js

```diff
@@ -0,0 +1,12 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
```

Comment:
Kindly add an `action creator` for `CHECK_STATUS` similar to what I explained on the `addBook` and `removeBook`.

You can do something similar to this:
```
export const checkStatus = (payload) => ({
  type: CHECK_STATUS,
  payload,
});
```